recipes/ckzg/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ckzg" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ckzg-{{ version }}.tar.gz
+  sha256: 4295acc380f8d42ebea4a4a0a68c424a322bb335a33bad05c72ead8cbb28d118
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - clang
```

Comment:
Why is `clang` listed here? You probably would either list it as a compiler or use `clangdev`.

Reply:
thanks. Most likely because it worked, and I didn't know that I could add it with compiler(). Interestingly, doing that meant it failed on Windows. So now I changed it to that but only use that line for `[lin]`.

Reply:
This should be unneeded as we have `{{ compiler('c') }}` already. Let's drop it

```suggestion
```